config/owasp/suppressions.xml

```diff
@@ -1,9 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress>
-    <gav regex="true">^.*jackson-databind.*$</gav>
-    <cve>CVE-2023-35116</cve>
-  </suppress>
   <suppress>
```

Comment:
Would you mind removing all suppressions in this file please? I've got a feeling none of them is needed anymore.
(I do appreciate that's out of scope of this work so happy for this to be merged as is too)